src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/SetAzureRmDataLakeAnalyticsCatalogSecret.cs

```diff
@@ -55,7 +55,7 @@ public class SetAzureDataLakeAnalyticsCatalogSecret : DataLakeAnalyticsCmdletBas
 
         [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = BaseParameterSetName, Position = 3,
             Mandatory = true, HelpMessage = "The host of the database to connect to in the format 'myhost.dns.com'.")]
-        public string Host { get; set; }
+        public string DatabaseHost { get; set; }
```

Comment:
Same comments - add an alais to prevent breaks, and beware of breaks in the output types